icons/tools/generators.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
do we want to disable this?

Reply:
We have in many of our other tools scripts but it does need this at the end:
```

/* eslint-enable no-console */
```

Reply:
Received a eslint error on the file. Most of the build.js files have it as they consist of console.log's